test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua

```diff
@@ -0,0 +1,97 @@
+--  Requirement summary:
```

Comment:
@istoimenova since these are smoke tests, these scripts should be in both branches I think. So I'll merge them into feature/extended_policy_functionality after closing this PR